macros/src/main/scala/me/rexim/morganey/meta/QuotationMacro.scala

```diff
@@ -0,0 +1,161 @@
+package me.rexim.morganey.meta
+
+import me.rexim.morganey.ast._
+
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.macros.whitebox.Context
+
+private[meta] class QuotationMacro(val c: Context) {
```

Comment:
Well, okay then. Our main focus would be on testing the expanded macros anyways.